tests/logger/test_log_entry.py

```diff
@@ -1101,601 +1103,946 @@ def test_deserialization_empty_fails(self, empty_set_signal_log_entry_as_dict):
+                == train_leave_block_section_log_entry_as_dict_serialized["message"]
             )
             assert (
-                str(resolve_fault_log_entry.track_speed_limit_fault_configuration)
-                == resolve_fault_log_entry_as_dict_serialized[
-                    "track_speed_limit_fault_configuration"
-                ]
+                str(train_leave_block_section_log_entry.run_id)
+                == train_leave_block_section_log_entry_as_dict_serialized["run_id"]
             )
             assert (
-                str(resolve_fault_log_entry.schedule_blocked_fault_configuration)
-                == resolve_fault_log_entry_as_dict_serialized[
-                    "schedule_blocked_fault_configuration"
-                ]
+                train_leave_block_section_log_entry.train_id
+                == train_leave_block_section_log_entry_as_dict_serialized["train_id"]
             )
             assert (
-                str(resolve_fault_log_entry.train_prio_fault_configuration)
-                == resolve_fault_log_entry_as_dict_serialized[
-                    "train_prio_fault_configuration"
+                train_leave_block_section_log_entry.block_section_id
+                == train_leave_block_section_log_entry_as_dict_serialized[
+                    "block_section_id"
                 ]
             )
             assert (
-                str(resolve_fault_log_entry.train_speed_fault_configuration)
-                == resolve_fault_log_entry_as_dict_serialized[
-                    "train_speed_fault_configuration"
+                train_leave_block_section_log_entry.block_section_length
+                == train_leave_block_section_log_entry_as_dict_serialized[
+                    "block_section_length"
                 ]
             )
 
         def test_deserialization_empty_fails(
-            self, empty_resolve_fault_log_entry_as_dict
+            self, empty_train_leave_block_section_log_entry_as_dict
         ):
-            """Test that Remove Fault Log Entry cannot be deserialized with no fields set."""
+            """Tests that TrainLeaveBlockSectionLogEntry cannot be deserialized
+            with no fields set."""
             with pytest.raises(ValidationError):
-                ResolveFaultLogEntry.Schema().load(
-                    empty_resolve_fault_log_entry_as_dict
+                TrainLeaveBlockSectionLogEntry.Schema().load(
+                    empty_train_leave_block_section_log_entry_as_dict
                 )
+
+
+class TestInjectFaultLogEntry:
+    """Tests for InjectFaultLogEntry."""
+
+    @pytest.fixture
+    def inject_fault_log_entry_as_dict(
+        self,
+        timestamp,
+        tick,
+        message,
+        run,
+        platform_blocked_fault_configuration,
+        track_blocked_fault_configuration,
+        track_speed_limit_fault_configuration,
+        schedule_blocked_fault_configuration,
+        train_prio_fault_configuration,
```

Comment:
I guess everything from here on are the parts that are shown as removed everywhere above?

Reply:
Please have a look at it again, I changed the identation of other classes, I accidentally changed